src/components/atoms/question/HeaderTest.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { View, Text, StyleSheet } from 'react-native'
+import { Icon } from 'native-base'
+import { Colors, Typography, Mixins } from '../../../styles'
+
+const HeaderTest = () => {
```

Comment:
QuestionHeader

Reply:
i think it is QuestionHeader

Reply:
Với nó là của cả bài mà, nó là Test chứ Minh nhỉ